aisuite/providers/sambanova_provider.py

```diff
@@ -0,0 +1,36 @@
+import os
+from aisuite.provider import Provider
+from openai import OpenAI
+
+
+class SambanovaProvider(Provider):
+    def __init__(self, **config):
+        """
+        Initialize the SambaNova provider with the given configuration.
+        Pass the entire configuration dictionary to the OpenAI client constructor.
+        """
+        # Ensure API key is provided either in config or via environment variable
+        config.setdefault("api_key", os.getenv("SAMBANOVA_API_KEY"))
+        if not config["api_key"]:
+            raise ValueError(
+                "Sambanova API key is missing. Please provide it in the config or set the SAMBANOVA_API_KEY environment variable."
+            )
+
+        # NOTE: We could choose to remove above lines for api_key since OpenAI will automatically
+        # infer certain values from the environment variables.
+        # Eg: OPENAI_API_KEY, OPENAI_ORG_ID, OPENAI_PROJECT_ID, OPENAI_BASE_URL, etc.
```

Comment:
This won't apply in this case, can you remove the comments?

Reply:
@ksolo Thanks for review, I removed these comments.